Makefile.toml

```diff
@@ -0,0 +1,98 @@
+#
+# Static checks
+#
+
+[tasks.format-check]
+workspace = false
+command = "cargo"
+args = ["fmt", "--", "--check"]
+
+[tasks.clippy]
+command = "cargo"
+args = ["clippy", "--all-targets", "--", "--deny=warnings"]
+
+[tasks.format-toml-check]
+description = "Check that all TOML files are formatted correctly"
+install_crate = { crate_name = "taplo-cli", version = "0.9.3", binary = "taplo", test_arg = "--help" }
+command = "taplo"
+args = ["format", "--check"]
+
+[tasks.doc-check]
+description = "Check documetation generation, similarly to how docs.rs will do it"
+command = "cargo"
+args = ["+nightly", "doc", "--all-features", "--no-deps"]
+env = { RUSTDOCFLAGS = "--cfg docsrs --deny=warnings" }
+install_crate = false
+
+#
+# Tests
+#
+
+[tasks.test-unit]
+command = "cargo"
+args = ["test", "--lib"]
+
+[tasks.test-integration]
+description = "Run integration tests"
+# This task is a placeholder, it will be overridden by the Makefile.toml in the crate
+workspace = true
+
+#
+# Tasks that modify content and are not run on CI
+#
+
+[tasks.format]
+workspace = false
+command = "cargo"
+args = ["fmt"]
+
+[tasks.proto]
+workspace = false
+command = "cargo"
+args = ["run", "--package", "tool_proto_build"]
+cwd = "crates/tool_proto_build"
+
+[tasks.format-toml]
+workspace = false
+install_crate = { crate_name = "taplo-cli", version = "0.9.3", binary = "taplo", test_arg = "--help" }
```

Comment:
Is there any way to make a dependency that installs taplo-cli so we don't have to ensure the version is the same in two locations?

Reply:
Yes https://github.com/holochain/kitsune2/pull/193/commits/bfac650a47793a6c3565597636c0609c664981e6